rpcs3/main.cpp

```diff
@@ -408,23 +413,25 @@ int main(int argc, char** argv)
 
 	// Command line args
 	QCommandLineParser parser;
+	parser.setOptionsAfterPositionalArgumentsMode(QCommandLineParser::ParseAsPositionalArguments);
 	parser.setApplicationDescription("Welcome to RPCS3 command line.");
-	parser.addPositionalArgument("(S)ELF", "Path for directly executing a (S)ELF");
-	parser.addPositionalArgument("[Args...]", "Optional args for the executable");
 
 	const QCommandLineOption help_option    = parser.addHelpOption();
 	const QCommandLineOption version_option = parser.addVersionOption();
-	parser.addOption(QCommandLineOption(arg_headless, "Run RPCS3 in headless mode."));
-	parser.addOption(QCommandLineOption(arg_no_gui, "Run RPCS3 without its GUI."));
-	parser.addOption(QCommandLineOption(arg_high_dpi, "Enables Qt High Dpi Scaling.", "enabled", "1"));
-	parser.addOption(QCommandLineOption(arg_rounding, "Sets the Qt::HighDpiScaleFactorRoundingPolicy for values like 150% zoom.", "rounding", "4"));
-	parser.addOption(QCommandLineOption(arg_styles, "Lists the available styles."));
-	parser.addOption(QCommandLineOption(arg_style, "Loads a custom style.", "style", ""));
-	parser.addOption(QCommandLineOption(arg_stylesheet, "Loads a custom stylesheet.", "path", ""));
-	const QCommandLineOption config_option(arg_config, "Forces the emulator to use this configuration file.", "path", "");
-	parser.addOption(config_option);
-	parser.addOption(QCommandLineOption(arg_error, "For internal usage."));
-	parser.addOption(QCommandLineOption(arg_updating, "For internal usage."));
+	parser.addOption(QCommandLineOption(arg_headless, QCoreApplication::tr("CommandLine", "Run RPCS3 in headless mode.")));
+	parser.addOption(QCommandLineOption(arg_no_gui, QCoreApplication::tr("CommandLine", "Run RPCS3 without its GUI.")));
+	parser.addOption(QCommandLineOption(arg_high_dpi, QCoreApplication::tr("CommandLine", "Enables Qt High Dpi Scaling."), "enabled", "1"));
+	parser.addOption(QCommandLineOption(arg_rounding, QCoreApplication::tr("CommandLine", "Sets the Qt::HighDpiScaleFactorRoundingPolicy for values like 150% zoom."), "rounding", "4"));
+	parser.addOption(QCommandLineOption(arg_styles, QCoreApplication::tr("CommandLine", "Lists the available styles.")));
+	parser.addOption(QCommandLineOption(arg_style, QCoreApplication::tr("CommandLine", "Loads a custom style."), "style", ""));
+	parser.addOption(QCommandLineOption(arg_stylesheet, QCoreApplication::tr("CommandLine", "Loads a custom stylesheet."), "path", ""));
+	parser.addOption(QCommandLineOption(arg_config, QCoreApplication::tr("CommandLine", "Forces the emulator to use this configuration file."), "path", ""));
+	parser.addOption(QCommandLineOption(arg_exit_after, QCoreApplication::tr("CommandLine", "Exit the application when the Emulator terminates.")));
+	parser.addOption(QCommandLineOption(arg_install_pkgs, QCoreApplication::tr("CommandLine", "Install pkg files before starting emulator."), "comma_separated_paths"));
```

Comment:
why comma seperated?
usually it's just spaces.
and then get the .values instead of .value

Reply:
My understanding is that spaces would not work with Positional Arguments, since how would it determine where the option list ends and the positional arguments start.
And I didn't see any obvious Qt examples which used options having multiple space separated values assigned to them.
This way 'works', has a non-ideal limitation, but certainly not a show stopper.